cfme/tests/cloud/test_tenant.py

```diff
@@ -104,9 +90,23 @@ def check_permissions(appliance, assigned_tenant):
     assert not view.toolbar.configuration.has_item('Manage Quotas')
 
 
+@pytest.fixture(scope="module")
+def product_features(appliance):
+    """Tree path navigation (for quota management permissions) to particular node of product feature
+     in RBAC for roles
+     """
+    config = "Settings,Configuration" if appliance.version < "5.11" else "Main Configuration"
+    product_feature = [
+        "Everything,{config},Access Control,Tenants,Modify,Manage Quotas,Manage Quotas ({tenant})".
```

Comment:
optional:
nice hack :) but I will try to avoid this for readability.
totally optional

Reply:
Yeah I'm not much of a fan of this either, using a string and then splitting it.  Consider just building the list directly.

```
product_feature = ['Everything']
product_feature.extend(['Settings', 'Configuration'] if appliance.version < '5.11' else ['Main Configuration])
product_feature.extend(['Access', 'others', Manage Quotas ({tenant})' for tenant in TENANTS])
```

Reply:
Also, this isn't that useful as a fixture, since there is no setup or teardown. 

This could be a regular method if used in more than one test, or just written into the test directly.

Reply:
Additionally it looks like `Settings` is now `User Settings` as well, at least in 5.10.5.

Reply:
Added in test itself.